lib/oxidized/nodes.rb

```diff
@@ -58,7 +58,8 @@ def show node
     def fetch node, group
       with_lock do
         i = find_node_index node
-        output = self[i].output.new
+        # Use first output type
+        output = self[i].output[0].new
```

Comment:
Not sure what you mean - you specify the outputs you want as a comma-separated list, the same way as with inputs:
```
output:
  default: git, file
```
So which is the default output? My simple solution was to use the first one for fetching.